src/main/java/seedu/address/logic/commands/CustomiseCommand.java

```diff
@@ -28,10 +34,23 @@ public class CustomiseCommand extends Command {
 
```

Comment:
`MESSAGE_USAGE` has not been updated for show and hide commands

Reply:
Ok, I have updated it